nncf/experimental/torch2/function_hook/hook_executor_mode.py

```diff
@@ -363,13 +366,16 @@ def execute_post_hooks(self, output: Any, op_meta: OpMeta) -> Any:
 
             if isinstance(output, list):
                 for idx, value in enumerate(output):
-                    output[idx] = self.hook_storage.execute_post_function_hooks(op_meta.op_name, idx, value)
+                    output[idx] = self.process_post_function_hooks_for_value(value, op_meta, idx)
```

Comment:
Question: why this change is only applicable for post hooks and does not apply for pre hooks?

Reply:
For pre hook need to not only register tensor and add edge, also need to process input arguments (tesnormeta) and add it to FunctionMeta